.github/workflows/build-docs.yml

```diff
@@ -1,15 +1,11 @@
 name: Build & Publish docs
 
 on:
-  pull_request:
-    paths:
-      - ".github"
-      - ".github/workflows/*-docs.yml"
   push:
+    branches:
+      - main
     paths:
-      - ".github"
-      - ".github/workflows/*-docs.yml"
-      - "docs/static/schemas/**"
+      - "docs/**"
```

Comment:
What happened in the workflows?

Reply:
well it was rebuilding on push / PR on the .github paths. I think it was leftover from prototyping but that doesn't make much sense really